index.js

```diff
@@ -15,9 +15,11 @@ function circuitBreakerPlugin (fastify, opts, next) {
   const threshold = opts.threshold || 5
   const timeoutErrorMessage = opts.timeoutErrorMessage || 'Timeout'
   const circuitOpenErrorMessage = opts.circuitOpenErrorMessage || 'Circuit open'
+  const onCircuitOpen = opts.onCircuitOpen
+  const onTimeout = opts.onTimeout
   const cache = lru(opts.cache || 500)
 
-  var routeId = 0
+  let routeId = 0
```

Comment:
the vars were replaced with `let` and `const` when I ran `standard --fix` to fix some of my formatting.  Happy to change back to `var` if that's preferred